src/components/tabs/tab-bar.tsx

```diff
@@ -165,24 +213,40 @@ const TabBar = (props: TabBarProps) => {
     }
     return false;
   };
-  const test = [...React.Children.map(props.children, item => item), ...addedTabs];
   return (
     <Fragment>
       <div className="bar__wrapper">
-      <ul className="tab__bar" ref={tabBar} >
-        {test.map((child: any) => {
+      <ul className="tab__bar"
+        onMouseMove={elementDrag}
+        onMouseLeave={closeDragElement}
+        ref={tabBar}
+      >
+        {tabList.map((child: any, i) => {
+          if (child.props.id !== 'new') {
```

Comment:
Please check the true first `=== 'new'` , inverting the returns

Reply:
this check is not being used anymore

Reply:
any unused code should be removed, then